packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -59,7 +59,7 @@ export const BreadcrumbItem: React.FunctionComponent<BreadcrumbItemProps> = ({
       {isDropdown && <span className={css(styles.breadcrumbDropdown)}>{children}</span>}
       {render && render({ className, ariaCurrent })}
       {to && !render && (
-        <Component href={to} target={target} className={className} aria-current={ariaCurrent}>
+        <Component {...props} href={to} target={target} className={className} aria-current={ariaCurrent}>
```

Comment:
Looks great.  Can you spread the props at the end?

Reply:
Done :+1: